Exchange-Guide.md

```diff
@@ -0,0 +1,196 @@
+# Exchange Guide (WIP)
+
+## Goal
+This guide describes how to setup `xud` with an existing exchange system and how to connect to the XUD Simulation Network, `xud-simnet` for short and do some test trades. `xud` is in alpha stage and the `xud-simnet` aims to give exchange operators a 'feel' of how things will work. `xud` in it's current stage should never be connected to a production system nor be used on mainnet. You'll be using test coins at all times. This guide purposely is kept relatively manual to make it understandable how `xud` functions. We are working on an improved setup using docker, which will also be the preferred way to use `xud` for exchanges in production.
+
+Once the setup of `xud-simnet` is completed, you will be able to query pending orders, place orders, and experiment with `xud`'s rich set of commands. Once a match is found in the network, your orders will be executed via cross-chain atomic swaps. `xud-simnet` currently supports atomic swaps between BTC and LTC. This guide will show how to to install GO, bitcoin and litecoin daemons, setup lightning daemons (LND) for bitcoin (BTC) and litecoin (LTC), setup `xud` and basic instructions of how to integrate its API and finally connect to the `xud-simnet`. This guide is written for Linux & MacOS operating systems and geared towards IT administrators of exchanges.
+
+## Describing the Setup
+To connect to `xud-simnet` you'll need to run several components which together provide support for exchanging order information and atomic swaps between BTC and LTC on the lightning network. You will setup the following components:
+- `BTCD` - full node, connected to the BTC chain
+- `LTCD` - full node, connected to the LTC chain
+- `LND-BTC` - lightning network daemon for the BTC network
+- `LND-LTC` - lightning network daemon for the LTC network
+- `XUD` - Exchange Union's decentralized exchange layer
+
+## Installation 
+### Prerequisites
+Make sure you have the below programs installed:
+- [git](https://gist.github.com/derhuerst/1b15ff4652a867391f03#file-linux-md)
+- [make](https://www.cyberciti.biz/faq/debian-linux-install-gnu-gcc-compiler/)
+- [node.js + npm](https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions)
+- python2.7 (preinstalled on most systems, check with `python --version`)
+- killall (`sudo apt-get install psmisc`)
+
+### Installing GO
+If GO is not yet installed on your system please use the following command to install it:
+```
```

Comment:
I also think it makes the markdown easier to read if there are blank lines before and after the ``` markers. This is actually not consistently the style in other pages, but I think it'd be good to start doing it.

Reply:
Done